### DISCUSSION TITLE
Jellyfin + Synology Surveillance Station = Extra Awesome!

### DISCUSSION THREAD

**Discussion by ianwestwood** [jellyfin/jellyfin#10003] [General]
Hey all,

I have jellyfin installed on my Synology DS920+ in a docker container.  Everything seems ticketyboo so far.  Kudos to everyone involved.... this is a great project.  Thank you for doing what you do.

Here's my question: Is there a way to add my Synology Surveillance Station recordings as a media library?  

I don't think there is a need for metadata support in this use case.  All the info I require is in the folder structure and filenames.  Here is a screenshot for example:

![Screenshot 2023-07-15 at 5 26 54 PM](https://github.com/jellyfin/jellyfin/assets/13510279/4dc7304b-f28b-4d51-b3a7-d4f2496696dd)

Ideally I'd like to be able to view by camera, date and time (files are MP4 both with & without audio).  

Being able to browse my timelapse videos (MP4 no audio) & snapshots (jpg) would be cool as well.  

Folder / file view would be ideal, but I can't figure out how to get to that view.  Does it exist?

Jellyfin appears to scan the files in the Surveillance folder (it took some hours to complete), and 'Surveillance Station' appears on my UI home page under my media along with everything else I might expect to see....  But when I try to view anything in 'Surveillance Station' I just see a blank page and spinning blue circle.

![Screenshot 2023-07-15 at 5 36 05 PM](https://github.com/jellyfin/jellyfin/assets/13510279/2faae89f-ffb0-4c4a-bc71-71202f194c29)

Any help or advice would be gratefully received.

Cheers!

**Comment by marcoh242** [comment]
You should check your permissions for this folder.

**Comment by sbsaylors** [comment]
Make sure that the permissions of your docker bind (I'm assuming bind) is good. You might also just as a test. Move most of the files to somewhere else (not in the docker bind) and leave like 1 file. Restart jellyfin docker and see if it does anything outside of spinning. 

Also, if you head into the docker log - you can see lots of info something. If you're using something like portainer - just head to the log here.
